### PR TITLE
isa-l: add a new package

### PR DESCRIPTION
**Pull request by hyoklee**
This is for https://github.com/spack/spack/pull/35649.

### CODE REVIEW

File: var/spack/repos/builtin/packages/isa-l/package.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    # Current
+    version("2.30.0", sha256="bcf592c04fdfa19e723d2adf53d3e0f4efd5b956bb618fed54a1108d76a6eb56")
+    version("2.29.0", sha256="832d9747ef3f0c8c05d39e3d7fd6ee5299a844e1ee7382fc8c8b52a268f36eda")
+    version("2.28.0", sha256="589202efdcfe437b1786750ec81bf93055e3b88a4bdf909d3b519f2a7134034b")
+    version("2.27.0", sha256="d398c5072e8e73bebf02ad4f1db3c13e27a7d96c8c1630e75a19c9bd79a92964")
+    version("2.26.0", sha256="938ccce1764ed8fb65a13b02295be5af9a5e0d91686efb7474bde666214153b3")
+    version("2.25.0", sha256="302bb38bf76be632dbd338ab97efe1c84d47dbe6265ff7af8cb373f256c84b48")
+
+    # Previous versions
+    # TODO
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/isa-l/package.py]: Do you still  need these comments?

**Review by tldahlgren — COMMENTED** [review]
Confirmed the branch exists and six version sha256.

File: var/spack/repos/builtin/packages/isa-l/package.py (hunk 1 above)
Comment (hyoklee) [reply var/spack/repos/builtin/packages/isa-l/package.py]: No. I removed them. Thank you!

**Review by tldahlgren — APPROVED** [review]
LGTM

**Pull request merged**